Ab#127307 expose additional fields for seafarer

TC-Cory opened the pull request.

AlexandreSanscartier commented on src/CSF.SRDashboard.Client/DTO/MpdisApplicantDto.cs:
```diff
@@ -39,6 +54,32 @@ public DateTime ParsedDateOfBirth
+
+        public string ParsedMinisiterCertificationExpirationDate => this.getParsedTime(this.MinisiterCertificationExpirationDate);
+
+        private string getParsedTime(long? longTime)
+        {
+            if (longTime.HasValue)
+            {
+                DateTime start = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+                return start.AddMilliseconds(longTime.Value).ToUniversalTime().ToString(Constants.MMM_DD_YYYY);
+            }
+            return Constants.NotAvailable;
+        }
```
Not a blocker, but we could consider moving this into a utility class for dates.

AlexandreSanscartier reviewed (approved):
Is there a reason we moved the properties from src/MPDIS.API.Wrapper/Services/MPDIS/Entities/FullApplicantInformation.cs to src/MPDIS.API.Wrapper/Services/MPDIS/Entities/TrimmedApplicantInformation.cs?

TC-Cory commented:
> Is there a reason we moved the properties from src/MPDIS.API.Wrapper/Services/MPDIS/Entities/FullApplicantInformation.cs to src/MPDIS.API.Wrapper/Services/MPDIS/Entities/TrimmedApplicantInformation.cs?

I believe I've moved the fields we would only need to expose instead of sending back the whole object. I can refactor this
